frontend/packages/erd-core/src/features/gtm/utils/pushToDataLayer.ts

```diff
@@ -0,0 +1,10 @@
+type WindowWithDataLayer = Window & {
+  dataLayer: Record<string, unknown>[]
+}
+
+declare const window: WindowWithDataLayer
+
+export const pushToDataLayer = (obj: Record<string, unknown>) => {
+  window.dataLayer = window.dataLayer || []
+  window.dataLayer.push(obj)
+}
```

Comment:
It's easy to clean up because we just need to delete the features/gtm directory and fix the compile errors! 👍🏻